pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -551,3 +559,58 @@ func (exp *flowExporter) sendDataSet() (int, error) {
 	klog.V(4).Infof("Data set sent successfully. Bytes sent: %d", sentBytes)
 	return sentBytes, nil
 }
+
+func fetchPodSubnets(k8sClient kubernetes.Interface) ([]*net.IPNet, error) {
+	podSubnets := []*net.IPNet{}
+	nodeList, err := k8sClient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		klog.Errorf("Failed to get node list from K8s: %v", err)
+		return podSubnets, err
+	}
+	for _, node := range nodeList.Items {
+		for _, podCIDR := range node.Spec.PodCIDRs {
+			_, subnet, err := net.ParseCIDR(podCIDR)
+			if err != nil {
+				klog.Errorf("Failed to parse subnet from CIDR string %s: %v", podCIDR, err)
+				return podSubnets, err
+			}
+			if !subnetInArray(podSubnets, subnet) {
+				podSubnets = append(podSubnets, subnet)
+			}
+		}
+	}
+	return podSubnets, nil
+}
+
+func subnetInArray(podSubnets []*net.IPNet, subnet *net.IPNet) bool {
+	for _, i := range podSubnets {
+		if i == subnet {
+			return true
```

Comment:
why not use i.Contains(subnet) to skip the subnet? i is not a great variable name in this context IMO.

Reply:
Thanks, net.IPNet.Contain function can only reports whether the network includes ip. I added some logic to judge whether the network includes the new network.